assets/js/app.js

```diff
@@ -81,7 +89,7 @@ channel.join()
         // console.log("Joined successfully", resp)
         console.log("Joined successfully") 
         my_id = resp.my_id
-        document.getElementById("my_id").textContent = my_id
+        document.getElementById("my_id").textContent = username
```

Comment:
If necessary, you can create another elements with the id of 'username' instead of using my_Id for username. Easy to miss stuff later on.